ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -50,6 +50,13 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: installer-rehearse-aws-ovn
```

Comment:
The rehearse yaml file is designed to validate the newly developing workflow, so you can freely update the existing test items, just replace workflow setting with yours, that will help reduce the number of rehearse jobs, unless several PRs are being developing at the same time, to avoid conflicts, have to introduce new test items.

Reply:
Great idea, so I don't need to add new test item for aws-ovn. let me try. Thanks

Reply:
wait for a moment, I am trying to update master branch's test items, maybe you can keep your test item for a while, once my commit is updated to upstream, you can sync and update it them.